clients/java/src/main/java/io/zeebe/client/impl/ZeebeClientImpl.java

```diff
@@ -103,10 +103,16 @@ public static ManagedChannel buildChannel(ZeebeClientConfiguration config) {
       throw new RuntimeException("Failed to parse broker contact point", e);
     }
 
-    // TODO: Issue #1134 - https://github.com/zeebe-io/zeebe/issues/1134
-    return ManagedChannelBuilder.forAddress(address.getHost(), address.getPort())
-        .usePlaintext()
-        .build();
+    final NettyChannelBuilder channelBuilder =
+        NettyChannelBuilder.forAddress(address.getHost(), address.getPort());
+
+    if (config.isSecureConnectionEnabled()) {
+      channelBuilder.useTransportSecurity().sslContext(config.getSslContext());
```

Comment:
If `config.getSslContext()` returns `null`, does still still work as expected?

Reply:
Yes, in that case the default value is used

Reply:
:+1: